EcatTxPdo txPdo
EcatSdoData sdo
EcatInfo info